.github/workflows/update-github.yaml

```diff
@@ -0,0 +1,45 @@
+name: "UpdateOctokit"
+
+on:
+  schedule:
+    - cron: '0 18 * * 0'
```

Comment:
can you add a comment in code, what this cron syntax means? According to [`https://crontab.guru/#0_18_*_*_0`](https://crontab.guru/#0_18_*_*_0) it means: "At 18:00 on Sunday"